sdk/helper/certutil/types.go

```diff
@@ -704,6 +704,45 @@ func (n NotAfterBehavior) String() string {
 	return "unknown"
 }
 
+type notBeforeBound int
```

Comment:
If this is just for constants for PKI, we can move it to `builtin/logical/pki`, but if we're looking to do enforcement in `certutil`, we could keep it here. 

I'm not really picky though, we can keep it here if we want.

Reply:
I don't mind, either. I'll move this to `builtin/logical/pki`

Reply:
Move the types to `builtin/logical/pki` instead.